Lib/pydoc.py

```diff
@@ -2675,6 +2678,9 @@ class BadUsage(Exception): pass
 {cmd} -k <keyword>
     Search for a keyword in the synopsis lines of all available modules.
 
+{cmd} -h <hostname>
```

Comment:
Using `-h` is a bit dodgy since it's so commonly used as shorthand for `--help`. Unfortunately I don't have a better suggestion beyond `-n`.